react/src/lib/components/ColorLegends/WebVizContinuousLegend.tsx

```diff
@@ -4,10 +4,15 @@ import { ContinuousLegend } from "@emerson-eps/color-tables";
 import { colorTablesArray } from "@emerson-eps/color-tables/";
 
 interface LegendProps {
-    title: string;
+    title?: string;
     min: number;
     max: number;
-    position?: number[] | null;
+    cssLegendStyles?: {
```

Comment:
I would think a more proper type for CSS would be `Record<string, unknown>` or `React.CSSProperties` (if this works with Dash). There are several more style options than the ones listed.